# origin: Debian, #241272
# cset: n/a
# inclusion: not submitted
# description: add TUNER_PHILIPS_NTSC binding for TD1536/D
# revision date: 2004-09-04

diff -urN kernel-source-2.4.26/drivers/media/video/bttv-cards.c kernel-source-2.4.26-1/drivers/media/video/bttv-cards.c
--- kernel-source-2.4.26/drivers/media/video/bttv-cards.c	2003-11-29 05:26:20.000000000 +1100
+++ kernel-source-2.4.26-1/drivers/media/video/bttv-cards.c	2004-04-03 18:05:15.000000000 +1000
@@ -2375,8 +2375,8 @@
         { TUNER_TEMIC_4039FR5_NTSC, "Temic 4039FR5" },
         { TUNER_PHILIPS_FQ1216ME,   "Philips FQ1216 ME" },
         { TUNER_TEMIC_4066FY5_PAL_I, "Temic 4066FY5" },
-        { TUNER_ABSENT,        "Philips TD1536" },
-        { TUNER_ABSENT,        "Philips TD1536D" },
+        { TUNER_PHILIPS_NTSC,        "Philips TD1536" },
+        { TUNER_PHILIPS_NTSC,        "Philips TD1536D" },
 	{ TUNER_PHILIPS_NTSC,  "Philips FMR1236" }, /* mono radio */
         { TUNER_ABSENT,        "Philips FI1256MP" },
         { TUNER_ABSENT,        "Samsung TCPQ9091P" },
